_includes/tutorials/error-handling/kstreams/markup/dev/run-dev-app.adoc

```diff
@@ -0,0 +1,13 @@
+Now that you have an uberjar for the Kafka Streams application, you can launch it locally.
+
+The application for this tutorial includes a record generator to populate the topic data.  Here is the list of records produced:
+
+`"All", "streams", "lead", "to", "Confluent", "Go", "to", "Kafka", "Summit"`
+
+
+Since we force some exceptions at different intervals while the streams application runs, you should see some stack traces in the console indicating an error, but the application will continue running.  However, when the application encounters an error that meets the threshold for max errors, it will shut down.
+
+No go ahead and run the following program, but watch the logs in the console and let the application run for a few seconds, you should observe it shutting down.
```

Comment:
Can you add a sample output from the logs that shows it shutting down?

Reply:
ack